css/global.css

```diff
@@ -900,13 +901,15 @@ Media Queries (overriding styles for different screen sizes)
 ===================================
 */
 
-/* custom breakpoint for top navigation 1360px*/
-@media screen and (max-width: 1360px) {
+@media screen and (max-width: 1395px) {
```

Comment:
custom mq to deal w nav link sizing and font-size

Reply:
In my experience, the navigation is usually where you have to break out some funky custom breakpoints.